protostar/argument_parser/argument_parser_facade.py

```diff
@@ -22,11 +26,66 @@
 )
 
 
+class InputFormat(Enum):
+    """
+    possible formats
+        0) --inputs 1 2 3
+        1) --inputs +a 1 +b 2 +c 3
+        2) --inputs a=1 b=2 c=3
+    """
```

Comment:
I can remove this but I think it improves readability

Reply:
It is fine, you can leave it, but plz apply reStructuredText formatting.

Reply:
I interactively rebased and removed this commit [handle 3 different formats](https://github.com/software-mansion/protostar/pull/1140/commits/463782d991b10f40e59539fd53cd071e3a971c72) as requested [here](https://github.com/software-mansion/protostar/pull/1140#issuecomment-1310028641) so this is no longer relevant.